dashboard/src/app/[tenantId]/userTaskDef/[...props]/components/Versions.tsx

```diff
@@ -1,3 +1,4 @@
+'use client'
```

Comment:
good catch putting this here even tho the parent component its client, this is best practice in-case this components parent changes to be a server component